### ISSUE TITLE
Builder travelling

### ISSUE THREAD

**Issue by Mailleweaver** [ldtteam/minecolonies#1216]
### Minecolonies version
0.8.2611
Problem may have begun in 0.8.1929 or 0.8.2488. Unsure which.

### Expected behavior
- Builder walks to travel around the colony.

### Actual behavior
- After playing with the mod for a while, the builder stops walking and instead stands in each location for a long time before eventually teleporting to the next destination.
- She may or may not walk around a build site after teleporting there, but she never walks to/from build sites, her hut, or her home.
- "Recall Worker," firing/hiring the same builder, hiring a different builder, and repairing the builder hut don't fix it.

### Steps to reproduce the problem
- Play for a while, building various buildings.
- I believe the problem started around the time I finished a level 4 wooden guard tower, but am not sure.
- I began play in 0.8.1929, started getting the hang of things (during which time the builder behaved as expected), started having problems, and somewhere in there the modpack I play with updated to 0.8.2488. I don't know if the problem started before or after the update. It has since updated to 0.8.2611, but the problem remains.

**Comment by Raycoms** [comment]
Is this in skyblock or in regular minecolonies?

**Comment by Mailleweaver** [comment]
Regular: Colonial Exploration modpack on Curse. My builder is now walking again in version 0.8.2797. I don't  know if it was an update that fixed it or my roads. I had Little Tiles blocks acting as stairs leading up to the builder hut and at a few more places around my colony, and Rustic slate blocks for the primary road material. Replacing the Little Tiles stairs with cobblestone stairs, replacing the slate with mossy cobble, and updating the mod are the only changes I've made in the time that she started walking again.

After a little testing, I've confirmed that it is the Little Tiles blocks that broke pathfinding. So, perhaps this should be considered a compatibility issue rather than a bug.
